cmd/rule-evaluator/internal/alerts.go

```diff
@@ -0,0 +1,24 @@
+package internal
```

Comment:
Can you add same license as in other files? I think this is why presubmit fails 🤗

Reply:
done